core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -244,7 +244,7 @@
       "If true, this node can participate in master election. In a multi-colo setup, turn this off "
       + "for clusters in the slave data center.";
   protected static final String MODE_MUTABILITY_DOC =
-      "If true, the mode of this node can be changed.";
+      "If true, the mode of this schema registry cluster can be changed.";
```

Comment:
Is this strictly true? What happens if I have two nodes with different settings for `mode.mutability`?

Reply:
Good point.  I've corrected it to be `If true, this node will allow mode changes if is the master.`

Reply:
Great, I really like that language.